source/includes/steps-connect-looker.yaml

```diff
@@ -0,0 +1,38 @@
+title: Add whitelist entries to Atlas for Looker.
+ref: add-whitelist-entries
+stepnum: 1
+content: |
+   `Add whitelist entries <https://docs.atlas.mongodb.com/security-whitelist/index.html>`_
+   to your Atlas cluster for all of `Looker's IP addresses <https://docs.looker.com/setup-and-management/enabling-secure-db>`_.
+---
+title: Connect Looker to your Atlas cluster.
+ref: connect-looker-to-atlas
+stepnum: 2
+content: |
+   
+   a. Log in to Looker.
+
+   #. Click :guilabel:`Admin`, then click :guilabel:`Connections.`
+
+   #. Click :guilabel:`New Connection`.
+
+   #. Enter the required settings in the :guilabel:`Connection Settings`
+      modal.
+
+      .. note::
+
+         Select :guilabel:`MongoBI` from :guilabel:`Dialect`. You can find
+         the other required settings in the Atlas :guilabel:`Connect` modal.
```

Comment:
Since we're in the BI docs, it might not be clear what the connect modal is. I also can't find a page that is worth linking to in the Atlas docs.

Reply:
I added a link to https://docs.atlas.mongodb.com/bi-connection/ in the note that references the Atlas Connect  modal. Fixed.